daemon/container_linux.go

```diff
@@ -999,3 +984,18 @@ func (container *Container) UnmountVolumes(forceSyscall bool) error {
 
 	return nil
 }
+
+func (container *Container) ExportRw() (archive.Archive, error) {
```

Comment:
It's just a move while I was working things out and debugging. This function is imminently about to be changed again as the Windows/Linux versions will diverge, but will be post this PR. I'll move it back